### PR TITLE
Solve some consistency problems

### PR DESCRIPTION
**Pull request by claucece**

### CODE REVIEW

File: draft-davidson-pp-protocol.md [hunk 1]
```diff
@@ -227,12 +226,12 @@ describe the structure of protocol data types and messages.
 ## Basic assumptions
 
 We make only a few minimal assumptions about the environment that the
-clients and servers that support the Privacy Pass protocol.
+clients and servers supporting the Privacy Pass protocol encounter.
```
Comment (alxdavids) [inline draft-davidson-pp-protocol.md:229]: I think the word `encounter` can be removed here.

File: draft-davidson-pp-protocol.md [hunk 2]
```diff
@@ -296,13 +295,13 @@ opaque PublicKey<1..2^32-1>;
 
 ### ServerUpdate {#pp-srv-cfg-update}
 
-The `ServerUpdate` struct contains the public information related to the
-creation of a new `ServerConfig` message. This is sent either directly
+The `ServerUpdate` struct contains the public information related to
+a new `ServerConfig` message from a server. This is sent either directly
```
Comment (alxdavids) [inline draft-davidson-pp-protocol.md:299]: `received from a server`?

File: draft-davidson-pp-protocol.md [hunk 3]
```diff
@@ -560,27 +557,28 @@ whether the data sent by the client is valid.
...
-- `ERR_MAX_EVALS`: Client attempted to invoke server issuance with
-  number of inputs that is larger than server-specified max_evals value.
-- `ERR_PROOF_VALIDATION`: Client unable to verify proof that is part of
-  the server response.
-- `ERR_DOUBLE_SPEND`: Indicates that a client has attempted to redeem a
-  token that has already been used for authorization.
+- `ERR_MAX_EVALS`: Error occurred when a client attempted to invoke
+  server issuance with a number of inputs that is larger than
+  server-specified max_evals value.
+- `ERR_PROOF_VALIDATION`: Error occurred when a client is unable to
```
Comment (alxdavids) [inline draft-davidson-pp-protocol.md:574]: requires minor rewording, perhaps: 
```
Error occurred when a client attempts to verify the proof...
```

File: draft-davidson-pp-protocol.md [hunk 4]
```diff
@@ -600,38 +598,38 @@ what is specified in {{I-D.irtf-cfrg-voprf}}.
...
 clients are aware of which configuration to use when interacting with
 the server.
 
-In situations where the number of clients are small, it could do this by
-sending the data to the client directly. But in situations where there
-is a large number of clients, the best way of doing is likely to be via
-posting this information to a public bulletin board. We assume that the
-server only has a single configuration in place at any one time. There
-are privacy restrictions related to this that are described in more
+In situations where the number of clients are small, a server perform
```
Comment (alxdavids) [inline draft-davidson-pp-protocol.md:605]: `perform` -> `performs`

File: draft-davidson-pp-protocol.md [hunk 5]
```diff
@@ -600,38 +598,38 @@ what is specified in {{I-D.irtf-cfrg-voprf}}.
...
+this action by sending the data to the client directly. But in situations
+where there is a large number of clients, the best way of doing is likely
```
Comment (alxdavids) [inline draft-davidson-pp-protocol.md:607]: `doing is` -> `doing this is`
(already existing typo)

**Comment by claucece** [comment]
Just changed them all ;)

**Pull request merged**